examples/complex/esp32/ios-gateway/extra/iOS_IoTCore_Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
I do not think you should be checking in files that are generated by the build step, remove this file.